apps/cookbook/src/app/showcase/slides-showcase/slides-showcase.component.html

```diff
@@ -36,14 +33,23 @@ <h2>Inherit height</h2>
   to each slide to make them equal height as seen in the example below.
 </p>
 
-<cookbook-iphone
-  viewMode="full-size"
-  [showViewModeToggle]="true"
-  src="/examples/slides-height/"
-  showExternalLink="true"
-></cookbook-iphone>
+<cookbook-example-viewer [html]="heightExample.template">
+  <div class="example-frame">
+    <cookbook-slides-height-example #heightExample></cookbook-slides-height-example>
+  </div>
+</cookbook-example-viewer>
+
+<h2>Custom heading</h2>
+
+<p>If you need a custom heading for the slides, you can add h1,h2,h3,h4,h5 or h6 tag.</p>
```

Comment:
This becomes slightly verbose, consider using 'any heading element' or `<h1>–<h6>` instead of listing all the tags, so it is easier to read.